src/platforms/android/configuration/integrations/okhttp.mdx

```diff
@@ -239,3 +239,102 @@ private final OkHttpClient client = new OkHttpClient.Builder()
   .addInterceptor(new SentryOkHttpInterceptor(new CustomBeforeSpanCallback()))
   .build();
 ```
+
+## HTTP Client errors
+
+HTTP Client errors such as bad response code are captured as error events and reported to Sentry. The error event will contain the `request` and `response` data such as `url`, `status_code`, and so on.
```

Comment:
Trying to avoid repetition. Should "bad response code" be in code font? Is it the actual error message you'd get?

```suggestion
HTTP client errors, like bad response code, are captured as error events and reported to Sentry. The error event will contain the `request` and `response` data, such as `url`, `status_code`, and so on.
```